### ISSUE TITLE
Issue while cloning the project on Windows 11

### ISSUE THREAD

**Issue by anooptp** [ubuntu/microk8s#2778]
On cloning the git repository on Windows 11, getting below error:

```
$ git clone https://github.com/ubuntu/microk8s.git
```
**Error**:
```
Cloning into 'microk8s'...
remote: Enumerating objects: 9649, done.
remote: Counting objects: 100% (1983/1983), done.
remote: Compressing objects: 100% (857/857), done.
remote: Total 9649 (delta 1280), reused 1716 (delta 1096), pack-reused 7666
Receiving objects: 100% (9649/9649), 15.00 MiB | 5.12 MiB/s, done.
Resolving deltas: 100% (6406/6406), done.
error: invalid path 'microk8s-resources/default-args/certs.d/localhost:32000/hosts.toml'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

**Comment by ktsakalozos** [comment]
Hi @anooptp thank you for bringing this to our attention.

The error you are seeing is because one of the files we checked in recently contained a `:` and this character is not allowed for filenames in windows. We have a PR under review to address this. The problem along with a workaround is described in https://stackoverflow.com/questions/5991805/how-do-i-clone-files-with-colons-in-the-filename

**Comment by anooptp** [comment]
Thank you @ktsakalozos for the update and a quick fix with PR #2781 .
